nilearn/decoding/decoder.py

```diff
@@ -1,43 +1,48 @@
 """High-level decoding object that exposes standard classification and
 regression strategies such as SVM, LogisticRegression and Ridge, with optional
-feature selection, integrated hyper-parameter selection
-and aggregation strategy in which the best models
-within a cross validation loop are averaged.
+feature selection, integrated hyper-parameter selection and aggregation strategy
+in which the best models within a cross validation loop are averaged.
+
+Also exposes a high-level method fREM which uses clustering and model
+ensembling to achieve state of the art performance
 """
 # Authors: Yannick Schwartz
 #          Andres Hoyos-Idrobo
 #          Binh Nguyen <tuan-binh.nguyen@inria.fr>
+#          Thomas Bazeille
 #
 # License: simplified BSD
 
 import itertools
 import warnings
 
 import numpy as np
+from joblib import Parallel, delayed
 from sklearn import clone
 from sklearn.base import RegressorMixin
 from sklearn.linear_model import LogisticRegression
 from sklearn.linear_model.base import LinearModel
 from sklearn.linear_model.ridge import RidgeCV, RidgeClassifierCV, _BaseRidgeCV
-from sklearn.model_selection import ParameterGrid, check_cv, LeaveOneGroupOut
+from sklearn.model_selection import ParameterGrid, check_cv, LeaveOneGroupOut, StratifiedShuffleSplit, ShuffleSplit
```

Comment:
Use parentheses for import line longer than 80 chars 
```suggestion
from sklearn.model_selection import (LeaveOneGroupOut, ParameterGrid,
                                     ShuffleSplit, StratifiedShuffleSplit,
                                     check_cv)
```